server/hedley/modules/custom/hedley_reports/hedley_reports.module

```diff
@@ -3912,6 +3914,346 @@ function hedley_reports_well_child_add_immunisation_activities(array $encounter_
+    hedley_reports_well_child_ecd_signs_from_18_months(),
+    hedley_reports_well_child_ecd_signs_from_2_years(),
+    hedley_reports_well_child_ecd_signs_from_3_years(),
+    hedley_reports_well_child_ecd_signs_from_4_years(),
+  ];
+}
+
+/**
+ * Returns the ECD signs for children aged 5 weeks.
+ *
+ * @return array
+ *   An array of ECD signs expected at 5 weeks.
+ */
+function hedley_reports_well_child_ecd_signs_from_5_weeks() {
+  return [
+    'follow-mothers-eyes',
+    'move-arms-and-legs',
+  ];
+}
+
+/**
+ * Returns the ECD signs for children aged 13 weeks.
+ *
+ * @return array
+ *   An array of ECD signs expected at 13 weeks.
+ */
+function hedley_reports_well_child_ecd_signs_from_13_weeks() {
+  return [
+    'raise-hands-up',
+    'smile',
+    'roll-sideways',
+  ];
+}
+
+/**
+ * Returns minor ECD signs for children aged 6 to 12 months.
+ *
+ * @return array
+ *   An array of minor ECD signs for 6 to 12 months.
+ */
+function hedley_reports_well_child_ecd_signs_6_to_12_months_minor() {
+  return [
+    'bring-hands-to-mouth',
+    'hold-head-without-support',
+    'hold-and-shake-toys',
+    'react-to-sudden-sounds',
+    'use-consonant-sounds',
+  ];
+}
+
+/**
+ * Returns major ECD signs for children aged 6 to 12 months.
+ *
+ * @return array
+ *   An array of major ECD signs for 6 to 12 months.
+ */
+function hedley_reports_well_child_ecd_signs_6_to_12_months_major() {
+  return [
+    'respond-to-sound-with-sound',
+    'turn-head-when-called',
+    'sit-without-support',
+    'smile-back',
+    'roll-tummy-to-back',
+    'reach-for-toys',
+  ];
+}
+
+/**
+ * Returns the ECD signs for children aged 15 months.
+ *
+ * @return array
+ *   An array of ECD signs expected at 15 months.
+ */
+function hedley_reports_well_child_ecd_signs_from_15_months() {
+  return [
+    'use-simple-gestures',
+    'stand-on-their-own',
+    'copy-during-play',
+    'say-mama-dada',
+    'can-hold-small-objects',
+  ];
+}
+
+/**
+ * Returns the ECD signs for children aged 18 months.
+ *
+ * @return array
+ *   An array of ECD signs expected at 18 months.
+ */
+function hedley_reports_well_child_ecd_signs_from_18_months() {
+  return [
+    'looks-when-pointed-at',
+    'use-single-words',
+    'walk-without-help',
+    'play-pretend',
+    'point-to-things-of-interest',
+  ];
+}
+
+/**
+ * Returns the ECD signs for children aged 2 years.
+ *
+ * @return array
+ *   An array of ECD signs expected at 2 years.
+ */
+function hedley_reports_well_child_ecd_signs_from_2_years() {
+  return [
+    'use-short-phrases',
+    'interested-in-other-children',
+    'follow-simple-instructions',
+    'kick-ball',
+    'point-at-named-objects',
+  ];
+}
+
+/**
+ * Returns the ECD signs for children aged 3 years.
+ *
+ * @return array
+ *   An array of ECD signs expected at 3 years.
+ */
+function hedley_reports_well_child_ecd_signs_from_3_years() {
+  return [
+    'dress-themselves',
+    'wash-hands-go-to-toiled',
```

Comment:
```suggestion
    'wash-hands-go-to-toilet',
```

Reply:
@AronNovak 

This typo origin is at ECD development somewhere at the end of 2021.
Since then, we have multiple occurrences in DB.
I don't think fixing now is worth the effort.